### ISSUE TITLE
cider-jack-in command uses unqalified name for nrepl

### ISSUE THREAD

**Issue by jr0cket** [clojure-emacs/cider#2943]
## Expected behaviour
When cider-jack-in-clj is started with a deps.edn configuration, the generated command line should included an unqualified nrepl dependency

/usr/local/bin/clojure -Sdeps '{:deps {nrepl/nrepl {:mvn/version "0.8.3"} cider/cider-nrepl {:mvn/version "0.25.4"}}}' -m nrepl.cmdline --middleware '["cider.nrepl/cider-middleware"]'

## Actual behaviour
When cider-jack-in-clj is started with a deps.edn configuration, the generated command line included an unqualified nrepl dependency

/usr/local/bin/clojure -Sdeps '{:deps {nrepl {:mvn/version "0.8.3"} cider/cider-nrepl {:mvn/version "0.25.4"}}}' -m nrepl.cmdline --middleware '["cider.nrepl/cider-middleware"]'

## Steps to reproduce the problem
Open any Clojure project with a deps.edn configuration file and run the cider-jack-in-clj command.

## Environment & Version information
Clojure CLI tools 1.10.697 and above

### CIDER version information

```
CIDER 1.0.0snapshot (package: 20201118.750)
openjdk version "11.0.9.1" 2020-11-04
Emacs 27.0.1
```

### Lein/Boot version
Not relevant

### Emacs version

Emacs 27.0.1

### Operating system

Ubuntu Linux 20.10

**Comment by bbatsov** [comment]
Didn't this fix solve it https://github.com/clojure-emacs/cider/commit/7aa71d79b0135a36b65a950c06ccfea1cd75178f ?

**Comment by jr0cket** [comment]
> Didn't this fix solve it [7aa71d7](https://github.com/clojure-emacs/cider/commit/7aa71d79b0135a36b65a950c06ccfea1cd75178f) ?

This change does look like the issue I raised.  I'll test when the next version of cider is release.
Thank you.
